src/styles/reset.css

```diff
@@ -0,0 +1,12 @@
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, sans-serif;
```

Comment:
Move to the html selector

Reply:
thats doesnt help, the font still differs from the mock

Reply:
![image](https://user-images.githubusercontent.com/117664824/236671091-7342d2ce-2094-4a4e-abd3-f1b325a29b56.png)